controls/V-72009.rb

```diff
@@ -1,55 +1,40 @@
 # encoding: utf-8
 #
-=begin
------------------
-Benchmark: Red Hat Enterprise Linux 7 Security Technical Implementation Guide
-Status: Accepted
-
-This Security Technical Implementation Guide is published as a tool to improve
-the security of Department of Defense (DoD) information systems. The
-requirements are derived from the National Institute of Standards and
-Technology (NIST) 800-53 and related documents. Comments or proposed revisions
-to this document should be sent via email to the following address:
-disa.stig_spt@mail.mil.
-
-Release Date: 2017-03-08
-Version: 1
-Publisher: DISA
-Source: STIG.DOD.MIL
-uri: http://iase.disa.mil
------------------
-=end
+
+# TODO this should use a `finding` array and we should make sure it is nil right?
 
 control "V-72009" do
   title "All files and directories must have a valid group owner."
-  desc  "Files without a valid group owner may be unintentionally inherited if a
-group is assigned the same Group Identifier (GID) as the GID of the files without a
-valid group owner."
+  desc  "Files without a valid group owner may be unintentionally inherited if
+a group is assigned the same Group Identifier (GID) as the GID of the files
+without a valid group owner."
   impact 0.5
-  tag "severity": "medium"
   tag "gtitle": "SRG-OS-000480-GPOS-00227"
   tag "gid": "V-72009"
-  tag "rid": "SV-86633r1_rule"
+  tag "rid": "SV-86633r2_rule"
   tag "stig_id": "RHEL-07-020330"
-  tag "cci": "CCI-002165"
+  tag "cci": ["CCI-002165"]
+  tag "documentable": false
   tag "nist": ["AC-3 (4)", "Rev_4"]
-  tag "check": "Verify all files and directories on the system have a valid group.
+  tag "check": "Verify all files and directories on the system have a valid
+group.
 
 Check the owner of all files and directories with the following command:
 
-Note: The value after -fstype must be replaced with the filesystem type. XFS is used
-as an example.
+Note: The value after -fstype must be replaced with the filesystem type. XFS is
+used as an example.
 
-# find / -xdev -fstype xfs -nogroup
+# find / -fstype xfs -nogroup
 
 If any files on the system do not have an assigned group, this is a finding."
-  tag "fix": "Either remove all files and directories from the system that do not
-have a valid group, or assign a valid group to all files and directories on the
-system with the \"chgrp\" command:
+  tag "fix": "Either remove all files and directories from the system that do
+not have a valid group, or assign a valid group to all files and directories on
+the system with the \"chgrp\" command:
 
 # chgrp <group> <file>"
+  tag "fix_id": "F-78361r1_fix"
 
-  describe command('find / -xdev -fstype xfs -nogroup') do
-    its('stdout.strip') { should match /^$/ }
+  describe command('find / -fstype xfs -nogroup') do
```

Comment:
This needs to loop through all valid filesystems on the system that are not `tmpfs` or `sysfs` (so far).